### ISSUE TITLE
Issue on handling eos when tensor_fn.create_attention_mask

### ISSUE THREAD

**Issue by AnselCmy** [RAGEN-AI/RAGEN#59]
Hi, thanks for the great work. I found that the eos token will be masked out when use tensor_fn.create_attention_mask. 
https://github.com/ZihanWang314/RAGEN/blob/4c457dc3a9509394c56acc97bb68e85cd19a4c75/ragen/llm_agent/tensor_helper.py#L35C1-L37C72
I don't know if this will have an effect on learning when to stop for models. Thanks!

**Comment by ZihanWang314** [comment]
Thanks for the nice catch! Will check!

**Comment by ZihanWang314** [comment]
In the current version, the problem has been fixed - the turns needed to learn will have a mask till the <im_end> which directs the model to stop output
